src/Core_Command.php

```diff
@@ -70,18 +70,18 @@ class Core_Command extends WP_CLI_Command {
 	 *
 	 * @subcommand check-update
 	 */
-	function check_update( $_, $assoc_args ) {
+	public function check_update( $_, $assoc_args ) {
 
 		$updates = $this->get_updates( $assoc_args );
 		if ( $updates ) {
-			$updates = array_reverse( $updates );
+			$updates   = array_reverse( $updates );
 			$formatter = new \WP_CLI\Formatter(
```

Comment:
Not directly related to this PR, but should a `use` statement be added for `WP_CLI\Formatter` and this line be changed to:
```suggestion
			$formatter = new Formatter(
```